libs/application/ui-fields/src/lib/VehicleRadioFormField/VehicleRadioFormField.tsx

```
@@ -0,0 +1,284 @@
+                          const fallbackMessage =
+                            (field.validationErrorFallbackMessage &&
+                              formatText(
+                                field.validationErrorFallbackMessage,
+                                application,
+                                formatMessage,
+                              )) +
+                            ' - ' +
+                            error.errorNo
+
+                          return (
+                            <Bullet>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)